### ISSUE TITLE
Add support for updating the firmware using the mobile app

### ISSUE THREAD

**Issue by mariobehling** [fossasia/badge-magic-android#987]
Propose and implement a way to update the firmware using the mobile app.

**Comment by Jhalakupadhyay** [comment]
as per my conversation with @kienvo I don't think it is possible to update the firmware directly from the app

**Comment by kienvo** [comment]
Yes, for now. This issue depends on fossasia/badgemagic-firmware#37.

**Comment by adityastic** [comment]
Duplicate of https://github.com/fossasia/badgemagic-app/issues/1205
